src/client/api/v1/get_balance.c

```diff
@@ -100,6 +100,16 @@ int get_balance(iota_client_conf_t const *conf, char const addr[], res_balance_t
     return -1;
   }
 
+  char const *cmd_balance = "";
+  if (addr_type == ED25519) {
+    cmd_balance = "/api/v1/addresses/ed25519/";
+  } else if (addr_type == BECH32) {
+    cmd_balance = "/api/v1/addresses/";
+  } else {
+    printf("[%s:%d]: get_balance failed (invalid address)\n", __func__, __LINE__);
+    return -1;
+  }
+
   // compose restful api command
   iota_str_t *cmd = iota_str_new(cmd_balance);
```

Comment:
`iota_str_new` allocate a dynamic buffer to compose the request. there is something we can do for avoiding dynamic memory, we know that
1) the length of the address string is fixed, 64 bytes for both ed25519 and bech32 representation. 
2) the longest command length is `/api/v1/addresses/ed25519/`, 27 bytes

therefore, we can use `snprintf` with a 92-byte buffer instead of `iota_str_new`.

Reply:
Done.